composer.json

```diff
@@ -27,11 +27,12 @@
       "url": "https://satis.wpteamhub.com"
     }
   ],
-  "require": {
-  },
+  "require": {},
   "require-dev": {
+    "johnpbloch/wordpress": "^5.0",
     "bluehost/wp-php-standards": "^1.0",
-    "phpunit/phpunit": "^7.4"
+    "phpunit/phpunit": "^6.5.14",
```

Comment:
Why did we step down in phpunit versions here?

Reply:
The WordPress unit testing framework doesn't work with version 7.4. I've heard it will work with 7.0, but figured rather than risk it I'd just go with the commonly used version.